packages/venia-ui/lib/drivers/index.js

```diff
@@ -1,4 +1,14 @@
-export { Link, Redirect, Route, Switch, withRouter } from 'react-router-dom';
+export {
+    Link,
+    Redirect,
+    Route,
+    Switch,
+    withRouter,
+    useLocation,
+    useHistory,
+    useParams,
+    useRouteMatch
```

Comment:
I don't think we need to define any of the hooks as `drivers` and we can probably remove `withRouter` too.